### DISCUSSION TITLE
Can not run tests on FullStack application

### DISCUSSION THREAD

**Discussion by Ioseb-1999** [DevSkillsHQ/discusssions#8] [Help] [answered]
This is how my package.json file looks like for starting up FullStack Application (According to the boilerplate)
{
  "name": "account-management-fullstack-level-2",
  "version": "1.0.0",
  "scripts": {
    "build:backend": "cd transaction-management-backend && npm install && npm run build",
    "build:frontend": "cd transaction-management-frontend && npm install && npm run build",
    "build": "npm run build:backend & npm run build:frontend",
    "start:backend": "cd transaction-management-backend && npm run start",
    "start:frontend": "cd transaction-management-frontend &&  npm run start",
    "start": "npm run start:backend & npm run start:frontend",
    "test": "cypress run"
  },
  "devDependencies": {
    "cypress": "^9.7.0"
  }
}

But It only starts backend side and how do I manage to start both of them at the same time? 

Backend port is - 5000
Frontend Port is - 3000

**Comment by fenske** [comment, marked as answer]
Hi @Ioseb-1999. I'm Anton from DevSkills. From what I can tell, you found a way around it with `&&` and `concurrently`. Good job! 

Let me know if I can assist in any other way.

I wish you the best of luck with the rest of your interview process!

**Reply by Ioseb-1999** [reply]
Hello @fenske Thanks.

Just in case if anyone will have same issue Here steps:

1.  Install concurrently  npm install concurrently --save
2.  Add this to the server folder in the package.json file in scripts section "start": "concurrently \"nodemon server.js\" \" cd ../path to frontend && npm run start\""

**Reply by fenske** [reply]
Thx a lot for sharing!

**Reply by alitanharo** [reply]
Hi, 
I am using fullstack-boilerplate-express-react  for this test and my app works 100% perfectly.  I have a problem running tests, I received this error
: The process '/usr/local/bin/npm' failed with exit code 1
I can't understand what the problem is, already tried all the previous solutions that exist in your community but they still don't work .
Can you Guide me, please?

**Reply by fenske** [reply]
Hi @alitanharo. Replied to you in https://github.com/orgs/DevSkillsHQ/discussions/3#discussioncomment-5893900
